package.json

```diff
@@ -40,9 +42,13 @@
   },
   "dependencies": {
     "ejs": "^3.1.9",
-    "fs-extra": "^11.2.0"
+    "fs-extra": "^11.2.0",
+    "globals": "^15.0.0"
   },
   "devDependencies": {
+    "@eslint/eslintrc": "^3.0.2",
```

Comment:
Don't need this here - there'll be a separate PR for updating the eslint config system

```suggestion
```